lighthouse-cli/test/cli/chrome-launcher-test.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+require('../../compiled-check.js')('chrome-launcher.js');
+
+const ChromeLauncher = require('../../chrome-launcher.js').ChromeLauncher;
+
+/* global describe, it */
+
+describe('ChromeLauncher', () => {
+  it('doesn\'t fail when killed twice', () => {
+    const chromeInstance = new ChromeLauncher();
+    chromeInstance.prepare();
+    chromeInstance.run();
+    return chromeInstance.waitUntilReady()
```

Comment:
`chromeInstance.run()` also [waits on `waitUntilReady()` internally](https://github.com/GoogleChrome/lighthouse/blob/a17fa72806317486bffe0862214477f363704be1/lighthouse-cli/chrome-launcher.ts#L147), so it isn't necessary to call `chromeInstance.waitUntilReady()` explicitly